### DISCUSSION TITLE
COPY project Class Library Project to Container

### DISCUSSION THREAD

**Discussion by vitorafgomes** [dotnet/sdk-container-builds#506] [Q&A]
When adding the Infrastructure project reference to Notification, when generating the Container, an error is returned.
Would it be necessary to make any configuration for COPY of projects?

![image](https://github.com/dotnet/sdk-container-builds/assets/5174765/dd0e77de-5024-426c-bde3-9b3bb280932c)

![image](https://github.com/dotnet/sdk-container-builds/assets/5174765/eb92b4a9-ec01-4ab6-bfe3-a9e8547b5ac9)

![image](https://github.com/dotnet/sdk-container-builds/assets/5174765/12eb7f29-7b9f-4fae-b077-7c2f5b39188b)

**Comment by baronfel** [comment]
My guess is you are publishing a solution here? If so you will just have to wait until .NET 8.0.100 RC2 - there was a bug in our MSBuild targets that caused projects with `IsPublishable` set to `false` (like Library projects) to attempt to publish. This doesn't make any sense to do.  There _might_ be a workaround - if you add the following dummy Target to your library projects you might be able to build:

```xml
<Target Name="PublishContainer" />
```

That will at least provide a target, even if it does nothing. Once RC2 releases and you update to it, you'd need to remove this.

**Reply by vitorafgomes** [reply]
Thanks for the great help.

**Comment by vitorafgomes** [comment]
All it took was adding the target, and the deployment worked.

![image](https://github.com/dotnet/sdk-container-builds/assets/5174765/bc38577e-e6cb-4d63-8842-ace33e0c0729)
